packages/experimental/Divider/src/Divider.tsx

```diff
@@ -35,7 +35,7 @@ export const Divider = compressible<DividerProps, DividerTokens>((props: Divider
   [tokens, cache] = patchTokens(tokens, cache, {
     flexAfter: props.alignContent === 'end' ? 0 : 1,
     flexBefore: props.alignContent === 'start' ? 0 : 1,
-    minHeight: props.vertical ? globalTokens.size240 : 0,
+    minHeight: props.vertical ? globalTokens.size200 : 0,
```

Comment:
\#nit can we bring the `props.vertical && hasContent` check down on line 68 and this logic together to evaluate the default minHeight in one place?

Reply:
"Yes we can" - Bob the Builder